src/server/DB/index.js

```diff
@@ -21,7 +21,7 @@ const pgp = require('pg-promise')(initOptions);
 require('dotenv').config();
 
 // cn
-const connectionString = "postgres://lzifmynw:hdBiND6R1SbP7oYgmqOUct4MTeAHa5K_@pellefant.db.elephantsql.com:5432/lzifmynw";
```

Comment:
this is gonna stay on gh history, definitely want to change user/pass on elephant at some point; otherwise anyone can get access to db from outside

Reply:
will do!